hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/upgrade/TestOMUpgradeFinalizer.java

```diff
@@ -249,7 +245,7 @@ private void setupVersionManagerMockToFinalize(
 
   private OMLayoutFeature mockFeature(String name, int version) {
     OMLayoutFeature f = mock(OMLayoutFeature.class);
-    when(f.name()).thenReturn(name);
+    Mockito.lenient().when(f.name()).thenReturn(name);
```

Comment:
Is there a reason for using lenient here and in the other places in this test? I think they are not needed

Reply:
Thank you Attila for explaining it to me. In that case this PR is looking good to me.